build.sbt

```diff
@@ -450,6 +467,9 @@ lazy val finiteState = crossProject(JVMPlatform, JSPlatform, NativePlatform)
   .jsSettings(
     scalaJSLinkerConfig ~= (_.withModuleKind(ModuleKind.CommonJSModule))
   )
+  .jsSettings(
+    scalaJSLinkerConfig ~= (_.withModuleKind(ModuleKind.CommonJSModule))
+  )
```

Comment:
```suggestion
```

as the exact same seems to be just above

Reply:
Merge conflict resolution went south 😅